safe-global/web-core
# fix: validate owners in parallel

## What it solves

Resolves #789

## How this PR fixes it

The `owners` group are now dependencies of each other. When one is validated, the others are as well.

## How to test it

1. Open Safe creation and navigate to the owner step.
2. Add the same owner twice and observe the error on the second field.
3. Edit the first owner entry and observe the error on the second field disappear.

## Screenshots

![owner validation](https://user-images.githubusercontent.com/20442784/194059748-d8f7ba45-fac8-43c0-8fa7-d5e7e5ffc8f9.gif)
